csharp/src/IceDiscovery/Lookup.cs

```diff
@@ -261,7 +277,7 @@ public void FindObjectById(string domainId, Identity id, ILookupReplyPrx? reply,
                     Debug.Assert(reply != null);
                     reply.FoundObjectByIdAsync(id, proxy);
                 }
-                catch (System.Exception)
+                catch (Exception)
```

Comment:
I think we could / should simply use:
```
try
{

}
catch
{
    // ignore
}
```
in this case. (Assuming we do indeed want to ignore all exceptions)

Reply:
Fixed this and a similar ones